packages/alfa-rules/src/sia-r62/serialise.ts

```diff
@@ -71,6 +72,33 @@ export namespace Serialise {
     );
   }
 
+  export function boxShadow(style: Style): string {
+    const boxShadow = style.computed("box-shadow").value;
+
+    if (Keyword.isKeyword(boxShadow)) {
+      return "";
+    }
+
+    const serializedShadows = [] as Array<string>;
```

Comment:
```suggestion
    const serializedShadows: Array<string> = [];
```
`as` is doing an actual cast while `:` is just a hint, and compiler is more likely to complain about a bad `:`.